tests/endtoend_tests/test_examples.py

```diff
@@ -760,6 +760,9 @@ class Config:
             ),
         ],
     ),
+    "tesseractreference": Config(  # Can't test requests standalone; needs target Tesseract. Covered in separate test.
+        test_with_random_inputs=False, sample_requests=[]
+    ),
```

Comment:
I think at some point we could rethink what are we putting in examples, and maybe allow some way to skip entries here if meaningful; can you create an issue about this?